src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaLocationConfig.java

```diff
@@ -10,20 +10,40 @@
 @PropertySource("${nakadi.config}")
 public class KafkaLocationConfig {
 
-    @Value("${nakadi.kafka.broker}")
-    private String kafkaAddress;
-
     @Value("${nakadi.zookeeper.brokers}")
     private String zookeeperAddress;
 
-    @Bean(name = "kafkaBrokers")
-    public String kafkaBrokers() {
-        return kafkaAddress;
-    }
+    @Value("${nakadi.zookeeper.kafkaNamespace}")
+    private String zookeeperKafkaNamespace;
+
+    @Value("${nakadi.exhibitor.address}")
+    private String exhibitorAddress;
+
+    @Value("${nakadi.exhibitor.port}")
+    private Integer exhibitorPort;
 
     @Bean(name = "zookeeperBrokers")
     public String zookeeperBrokers() {
         return zookeeperAddress;
     }
 
+    @Bean(name = "zookeeperKafkaNamespace")
+    public String zookeeperKafkaNamespace() {
+        return zookeeperKafkaNamespace;
+    }
+
+    @Bean(name = "exhibitorAddress")
+    public String exhibitorAddress() {
+        return exhibitorAddress;
+    }
+
+    @Bean(name = "exhibitorPort")
+    public Integer exhibitorPort() {
+        return exhibitorPort;
+    }
```

Comment:
I personally would prefer using constructor based injection instead having all those strings as beans for configuration. Makes it more straightforward, better testable and clear.